### PR TITLE
started added functions

### PR DESCRIPTION
**Pull request by SimoneOdegard**

### CODE REVIEW

File: src/AddGenre.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      <>
+      <Button onClick={this.displayModal}>Add Genre</Button>
+      <Modal show={this.state.displayModal}>
+        <Modal.Dialog>
+          <Modal.Header closeButton>
+            <Modal.Title>Add Genre</Modal.Title>
+          </Modal.Header>
+
+          <Modal.Body>
+            <Form >
```
Comment (ganttArt) [inline src/AddGenre.js:37]: Might want to consider moving the form into a new component when refactoring.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Header from './Header';
+import Login from './Login';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route
+} from "react-router-dom";
+import Display from './Display';
```
Comment (ganttArt) [inline src/App.js:9]: What is Display? This is not a very understandable name.

File: src/Login.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Card from 'react-bootstrap/Card';
+import LoginButton from './LoginButton';
+
+class Login extends React.Component {
+  render() {
+    return(
+      <>
+        <center>
```
Comment (ganttArt) [inline src/Login.js:10]: what does having center here do?

**Review by ganttArt — APPROVED** [review]

**Review by JessiVelazquez — APPROVED** [review]
Nice work

**Pull request merged**